src/containers/table/Table.js

```diff
@@ -50,23 +50,28 @@ class TableView extends React.Component {
         super(props);
         this.state = {
             sortBy: 'name',
-            order: null,
+            order: 'desc',
             showCount: 20,
-            unSelectedTags: props.tags,
+            unSelectedTags: props.tags.sort(),
```

Comment:
sort() is case sensitive. don't know if data allows us to presume that all tags are lowercased